Mathlib/Analysis/SpecialFunctions/Integrals.lean

```diff
@@ -3,7 +3,10 @@ Copyright (c) 2021 Benjamin Davidson. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Benjamin Davidson
 -/
+import Mathlib.Analysis.Meromorphic.FactorizedRational
```

Comment:
When you need to add a lot of imports to a file, it's a sign that in fact you should instead create a new file.

Reply:
Implemented.